test/unit/DrbbContainer.test.js

```diff
@@ -13,33 +13,10 @@ describe('DrbbContainer', () => {
   let component;
   const context = mockRouterContext();
 
-  describe('initial render', () => {
```

Comment:
Would it be good to add back a test that simulates the results loading, so that these checks can still run?